### DISCUSSION TITLE
Upgrade to a newer docker container of kibana. What about custom changes in Kibana?

### DISCUSSION THREAD

**Discussion by sleepingbel** [Security-Onion-Solutions/securityonion#3060] [Unsupported Versions] [answered]
Hello All,

I have created two extra dashboards (Firewall & VPN) in Kibana and modified the syslog dashboard visualizations.
When there is a newer docker container of Kibana, do you need to create those dashbords and visualizations again?
Ore are these placed on de master and will salt configure this again?

Regards

Bart

**Comment by weslambert** [comment, marked as answer]
As long as you create custom dashboards and copy the existing visualizations to be used in those custom dashboards, then your changes will be saved.  Otherwise, they will be overwritten.

**Reply by sleepingbel** [reply]
Thx Wes
